src/index.js

```diff
@@ -36,7 +37,15 @@ const applyConfig = (config) => {
     config.settings.errorHandlers.push(errorHandler);
   }
   config.settings.sentryOptions = sentryOptions;
-  config.settings.storeExtenders = (stack) => [crashReporter, ...stack];
```

Comment:
I don't like this style. Is it because it's an anon function? then define it here. Better to leave the crashReporter as a simple middleware, un-aware of the volto configuration mechanism, with the storeExtenders.